src/base/preferences.cpp

```diff
@@ -61,6 +60,30 @@
 
 namespace
 {
+    template <typename T>
+    T value(const QString &key, const T &defaultValue = {})
```

Comment:
Are these overloads (with QString key) used in this file?

Reply:
yes `setSpeedWidgetGraphEnable()`